package.json

```diff
@@ -21,7 +21,7 @@
     "supertest": "^3.1.0",
     "test-all-versions": "^4.1.0",
     "tslint": "^5.8.0",
-    "typeorm": "^0.1.4",
+    "typeorm": "^0.2.34",
```

Comment:
soft delete is already impl in typeorm, but we need to upgrade to 0.2.3+. ok to upgrade to latest?